src/modules/role/schemas/Role.js

```diff
@@ -0,0 +1,17 @@
+import { ADMIN, HIL } from '../../../constants/Role'
+import { Entity, Id, String } from '../../../core/schemas'
+import UserRole from '../../user_role/schemas/UserRole'
+
+const Role = {
+  collectionName: 'Roles',
+  idField: 'roleName',
+  name: 'Role',
+  parentRefIdField: 'userId',
+  parentSchema: UserRole,
+  schema: Entity.schema.keys({
+    userId: Id.schema.required(),
+    roleName: String.schema.required()
```

Comment:
Can limit this with the allow statement to only be the enumerable values of Role

Reply:
I imported them but then didn't use them 😅